Scanner.rules

```diff
@@ -0,0 +1,38 @@
+MainRule  "str:havij" "msg:Havij-SQL_scanner" "mz:$HEADERS_VAR:User-Agent" "s:$UWA:8" id:42000312  ;
+MainRule  "str:http://http://" "msg:Abnormal double http:// in HTTP header," "mz:HEADERS" "s:$UWA:8" id:42000310  ;
+# http://pastebin.com/NP64hTQr# http://blog.initiative-s.de/2013/09/kompromitierte-wordpress-blogs-werden-fuer-ddos-attacken-genutzt/
+# neu su dung wp thi tat rule nay
```

Comment:
Please write this comment in english: `If using wp then turn off this rule`

Reply:
Thanks for the comment, I apologize for the issue.
This is if you are using wordpress code please disable it. The reason is that wordpress has a cron prefix that will be blocked if using these rules.